### PR TITLE
Fixed 981 and 975

### PR DESCRIPTION
**Pull request by vshand11**
1) Fixed crdcdh-981 and 975 for message updating.
2) implemented 1161 and 1136

### CODE REVIEW

File: src/file_validator.py [hunk 1]
```diff
@@ -312,7 +318,7 @@ def validate_all_files(self, submission_id):
                     file_batch = self.mongo_dao.find_batch_by_file_name(submission_id, "data file", file_name)
                     batchID = file_batch[ID] if file_batch else "-"
                     displayID = file_batch["displayID"] if file_batch else None
-                    msg = f'File “{file_name}”: no record found.'
+                    msg = f'File “{file_name}”: associated metadata not found. Please upload associated metadata (aka. manifest) file'
```
Comment (n2iw) [inline src/file_validator.py:321]: Please also change the error title to "Orphaned file found". Why I remember we've done it?

File: src/file_validator.py [hunk 2]
```diff
@@ -234,8 +234,14 @@ def validate_file(self, fileRecord):
             }
             self.mongo_dao.save_file_md5(md5_info)
 
-        if int(org_size) != int(size) or org_md5 != md5:
-            msg = f'File “{file_name}”: file integrity check failed.'
+        if int(org_size) != int(size):
+            msg = f'File “{file_name}”: expected size: {org_size}, actual size: {size}.'
+            self.log.error(msg)
+            error = create_error("Data file Integrity check failed", msg)
```
Comment (n2iw) [inline src/file_validator.py:240]: Please also change the title of the error, it's in the comment in the ticket

File: src/file_validator.py [hunk 3]
```diff
@@ -234,8 +234,14 @@ def validate_file(self, fileRecord):
...
+            return STATUS_ERROR, error
+        
+        if org_md5 != md5:
+            msg = f'File “{file_name}”: expected MD5: {org_md5}, actual MD5: {md5}.'
             self.log.error(msg)
             error = create_error("Data file Integrity check failed", msg)
```
Comment (n2iw) [inline src/file_validator.py:246]: Please also change the title of the error, it's in the comment in the ticket

**Review by n2iw — CHANGES_REQUESTED** [review]

File: src/file_validator.py (hunk 1 above)
Comment (vshand11) [reply src/file_validator.py:321]: updated

File: src/file_validator.py (hunk 2 above)
Comment (vshand11) [reply src/file_validator.py]: updated

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**